documentation/doc.css

```diff
@@ -119,6 +119,11 @@ html {
     margin: 1rem 2rem;
 }
 
+/* nav-bar component */
+.margin-nav{
+    margin: 2rem;
+}
```

Comment:
Why have you given nav-bar css in doc.css when you have nav-bar-component.css

Reply:
that nav bar for documentation page.